scripts/lint-java.js

```diff
@@ -0,0 +1,198 @@
+// code taken from facebook/react-native repo
```

Comment:
Please add a link with commit to allow comparing them in the future.
In Github, you can click the "y" key to change the URL with the latest changes into a commit-based one.